docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst

```diff
@@ -104,7 +104,7 @@ Bareos :sinceVersion:`15.2.0: requires: jansson` requires the :ref:`Jansson libr
    # DIST=Fedora_26
    # DIST=Fedora_25
```

Comment:
These DISTs are not the distributions delivered with Bareos 19.2. Same applies to the other sections.

Reply:
I modified the shell script examples to use recent dist versions, thanks for the review.